docs/creating/index.md

```diff
@@ -6,14 +6,24 @@ The model specifies one or more **nodes** along with the technologies allowed at
 **Demand technologies** remove carriers from the system through a **sink**, while **storage technologies** can store carriers at a specific node.
 Putting all of these possibilities together allows a modeller to specify as simple or as complex a model as necessary to answer a given research question.
 
+![Visual description of the Calliope terminology.](../img/description_of_system.svg)
+
 ??? info "An overview of the Calliope terminology"
-    The terminology defined here is used throughout the documentation and the model code and configuration files:
+    The terminology defined here is used throughout the documentation, and the model code and configuration files:
```

Comment:
I would not hide this section with `???`. This is quite important, and users might miss it.